pkg/common/consumer/consumer_handler.go

```diff
@@ -52,12 +53,32 @@ func WithSaramaConsumerLifecycleListener(listener SaramaConsumerLifecycleListene
 	}
 }
 
+// WithInflightRequestInterrupt configures the request timeout. Default is set to 60s.
```

Comment:
wrong comment `WithInflightRequestInterrupt` for the time out

Reply:
you are fast!